src/pages/RecoverPasswordPage.tsx

```diff
@@ -59,7 +59,7 @@ const RecoverPasswordPage = () => {
           handleChange={handleChange}
```

Comment:
Remember that the handleChange property is no longer sent but directly onChange

Reply:
Ok I review in a moment and see what fix I can apply.